spec/support/ruby_version.rb

```diff
@@ -1,5 +1,8 @@
-def mri22?
-  ruby_version.start_with?('ruby_2.2')
+# For things Broken in Ruby <= 2.2.2, Fixed in 2.2.3+
+def pending_for_mri220_to_mri222
+  version = ruby_version
+  return unless version.start_with?('ruby_2.2') && version.end_with?('0', '1', '2')
```

Comment:
There's a finate list of these, can we just check for something like `['ruby_2.2....].include?`.